composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
@keradus My point wasn't about weigth of packages. The point is that I've seen too many times a project that offers 100 functionalities, you want to use 1 of them, and you end up with an entire operative system to use just a hundredth of it.

For example, Zend has already taken this approach: [zend-view](https://github.com/zendframework/zend-view/blob/master/composer.json) **can** operate dozens of plugin, and they are all in `require-dev`, but you are free to choose which one to install.
The [zend-cache](https://github.com/zendframework/zend-cache/blob/master/composer.json) does this with extensions too.